src/main/java/training/bookish/controllers/LoansController.java

```diff
@@ -0,0 +1,26 @@
+package training.bookish.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import training.bookish.models.LoanForm;
+import training.bookish.services.LoanService;
+
+@Controller
+@RequestMapping("/loans")
+public class LoansController {
+
+    @Autowired
+    private LoanService loanService;
+
+    @GetMapping
+    public String getBooks(){
+        return "redirect:/";

```

Comment:
Just to check - is this a work in progress?

Reply:
Yes, full implementation will be in the next PR.